File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int pnum=0, ynum=0;
        
        for(int i=0; i<s.length(); i++){
            if(s.charAt(i)=='p' || s.charAt(i)=='P') pnum++;
            else if(s.charAt(i)=='y' || s.charAt(i)=='Y') ynum++;
        }

        if(pnum!=ynum) answer=false; 
        
        return answer;
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    var answer = true;
    s = s.toUpperCase();
    let pn=0, yn=0;
    
    for(let i=0; i<s.length; i++) {
        if(s[i]==="P") pn++;
        else if (s[i]==="Y") yn++;
        else continue;
    }
    
    if(pn!=yn) answer=false;

    return answer;
}